src/main/java/seedu/academydirectory/ui/HelpWindow.java

```diff
@@ -34,7 +43,33 @@
      */
     public HelpWindow(Stage root) {
         super(FXML, root);
-        helpMessage.setText(HELP_MESSAGE);
+        StringBuilder builder = new StringBuilder(HELP_MESSAGE);
+        MarkdownView markdownView = new MarkdownView();
+        try {
```

Comment:
Hmm, this try-catch block could be refactored into a small function so it looks cleaner :grin: